db/projects/blockmason-credit-protocol/project.yaml

```diff
@@ -0,0 +1,57 @@
+name: BlockMason Credit Protocol
```

Comment:
I think you do not have correct logo, the correct should be this:
https://pbs.twimg.com/profile_images/966456050869182465/1aN6Luy__400x400.jpg

Source: https://github.com/blockmason

Reply:
Anyway, still, the logo you have used is on ehterscan and coinmarketcap, but it does not mean that its correct. From their web page and github, they use different logo. We should use the current one.